### PR TITLE
added WC/Untamed cards - added growth counter related action wrappers.

### PR DESCRIPTION
**Pull request by Achocholacek**

### CODE REVIEW

File: server/game/cards/03-WC/DeepwoodDruid.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const Card = require('../../Card.js');
+
+class DeepwoodDruid extends Card {
+    setupCardAbilities(ability) {
+        this.play({
+            reap: true,
+            target: {
+                cardType: 'creature',
+                optional: false,
```
Comment (cryogen) [inline server/game/cards/03-WC/DeepwoodDruid.js]: Optional: false is the default so this is redunant

File: server/game/cards/03-WC/SongOfTheWild.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const Card = require('../../Card.js');
+
+class SongOfTheWild extends Card {
+    setupCardAbilities(ability) {
+        this.play({
+            effect: 'give each friendly creature \'Reap: Gain 1A\' for the remainder of the turn',
```
Comment (cryogen) [inline server/game/cards/03-WC/SongOfTheWild.js]: A - >amber

Comment (Achocholacek) [reply server/game/cards/03-WC/SongOfTheWild.js]: fixed in latest pr.

File: server/game/cards/03-WC/DeepwoodDruid.js (hunk 1 above)
Comment (Achocholacek) [reply server/game/cards/03-WC/DeepwoodDruid.js]: fixed in latest pr.

**Comment by Achocholacek** [comment]
should now be in line with expectations

**Pull request merged**